message/test_html.py

```diff
@@ -0,0 +1,60 @@
+from .models import Message
+from daycare.models import DayCare
+import pytest
+import random
+
+
+@pytest.mark.django_db()
+class TestMessageHtml:
+    def test_shown_only_relevant_contacts(self, client):
+        daycare = random.choice([daycare for daycare in DayCare.objects.all() if 'static' in daycare.user.username])
+        client.force_login(user=daycare.user)
```

Comment:
overall in tests, I think you cant use random methods, so if the test fails or succeeds you cant detect what made the reason for the tests to succeed/fail, therefor its should be explicit.

Reply:
Tests should never have a random element in them. you should test all the **different** relevant possibilities explicitly. you should be able to depend on the test results.

Reply:
Thanks for the good reviews guys!
removed random, now we test all users and all their contacts